Makefile

```diff
@@ -108,8 +108,10 @@ clean: ## clean up binaries
 	@echo "🐳 $@"
 	@rm -f $(BINARIES)
 
-install: $(BINARIES) ## install binaries
-	@echo "🐳 $@"
+install: ## install binaries
+	@ls $(BINARIES) > /dev/null 2>&1 || \
```

Comment:
As discussed in https://github.com/docker/containerd/pull/432#issuecomment-273583290 ,
it can depend on non-go files as well. So I think we should not use dependency list at the moment.